log/formatters_test.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+import { assertEquals } from "../assert/mod.ts";
+import { jsonFormatter } from "./formatters.ts";
+import { LogRecord } from "./logger.ts";
+
+const makeRecord = (msg: string, args: unknown[] = []) =>
+  new LogRecord({
+    msg,
+    args,
+    level: 20,
+    loggerName: "user-logger",
+  });
+
+Deno.test("jsonFormatter with just a message", function () {
+  const result = JSON.parse(jsonFormatter(makeRecord("User service exploded")));
+
+  assertEquals(result.level, "INFO");
+  assertEquals(typeof result.datetime, "number");
+  assertEquals(result.message, "User service exploded");
+  assertEquals(Object.keys(result), ["level", "datetime", "message"]);
```

Comment:
If I mock the time then I can write much nicer assertions, I'm just waiting for https://github.com/denoland/deno_std/pull/3961 to land so the test look pretty.